src/main/java/org/homework/domain/Todo.java

```diff
@@ -20,15 +26,23 @@ public String getContent() {
         return content;
     }
 
+    public LocalDate getDeadline() {
+        return deadline;
+    }
+
     public boolean isCompleted() {
         return completed;
     }
 
-    public void setContent(String content) {
-        this.content = content;
+    public String getCompletedStr() {
+        return completed ? "완료" : "미완료";
     }
 
     public void setCompleted(boolean completed) {
         this.completed = completed;
     }
```

Comment:
요구사항에서는 완료로 변경하는것만 있기 때문에 setMethod는 필요하지 않습니다.
setMethod가 존재하면, 어디선가 완료에서 미완료로 변경할 수 있는 가능성이 존재하게 됩니다.
객체지향에서는 getter,setter를 지양해야하는데요, 클래스의 상태를 외부가 아닌 내부에서 변경할 수 있도록 하는게 핵심입니다.
```suggestion
    public void complete() {
        this.completed = true;
    }
```